examples/inputs/example_02a/config.yaml

```diff
@@ -12,12 +12,12 @@ base:
   learning_config:
     observation_dimension: 50
     action_dimension: 2
-    continue_learning: False
-    load_model_path: None
+    continue_learning: True
+    load_model_path: C:\Users\tg3533\assume\examples\inputs\learned_strategies\example_02a_base
```

Comment:
also having an absolute path for the policies loading is not ideal, please use relative paths